packages/nuxt/src/core/nuxt.ts

```diff
@@ -621,4 +628,15 @@ export async function loadNuxt (opts: LoadNuxtOptions): Promise<Nuxt> {
   return nuxt
 }
 
+async function checkDependencyVersion(name: string, nuxtVersion: string): Promise<void> {
+  const path = await resolvePath(name).catch(() => null)
+
+  if (!path) return
+  const { version } = await readPackageJSON(path)
+
+  if (version !== nuxtVersion) {
+    console.error(`Version mismatch for ${name} and nuxt: expected ${nuxtVersion} (nuxt) but got ${version}`)
```

Comment:
Can we somehow hint people what to do on a version mismatch?

Reply:
Do you have any specific message in mind ? Should we say to check the package.json (or more ) ?

Reply:
@huang-julien Maybe ask to check the package.json and/or refresh the lockfile? 🤔